Documentation/user-guides/getting-started.md

```diff
@@ -343,6 +343,34 @@ spec:
 
 This enables the frontend team to create new ServiceMonitors and Services which allow Prometheus to be dynamically reconfigured.
 
+## Include PodMonitors
+
+Finally, a Prometheus object defines the `podMonitorSelector` to specify which PodMonitors should be included. Above the label `team: frontend` was specified, so that's what the Prometheus object selects by.
+
+[embedmd]:# (../../example/user-guides/getting-started/prometheus.yaml)
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  name: prometheus
+spec:
+  serviceAccountName: prometheus
+  serviceMonitorSelector:
+    matchLabels:
+      team: frontend
+  podMonitorSelector:
+    matchLabels:
+      team: frontend
```

Comment:
This part is generated by `embedmd` from https://github.com/coreos/prometheus-operator/blob/master/example/user-guides/getting-started/prometheus.yaml. Modify it first and regenerate files (`make generate`).